src/nni_manager/training_service/pai_base/paiYarn/paiYarnTrainingService.ts

```diff
@@ -471,9 +362,9 @@ class PAITrainingService implements TrainingService {
         return deferred.promise;
     }
 
-    private async statusCheckingLoop(): Promise<void> {
+    protected async statusCheckingLoop(): Promise<void> {
```

Comment:
base class already has similiar one, should not be re-implemented like this.

Reply:
fixed.